boilerplate/openshift/golang_osd_cluster_operator/ensure.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
This is a cool idea. It would be neat to see as much of it as possible made part of [_lib](https://github.com/openshift/boilerplate/tree/master/boilerplate/_lib) and just have the configuration (some kind of declarative list of dep/ver/loc) in the convention subdir in a file with a known name.

Reply:
Gah, ignore this comment. I was thinking of this completely wrong. Your intent was (correctly) to run this as part of the `make` target, not to run it during `update`.